tasks/recover/register_template.yml

```diff
@@ -0,0 +1,22 @@
+- name: Register unregistered Template
+  ovirt_templates:
+      state: registered
+      storage_domain: "{{ storage.name }}"
+      id: "{{ unreg_template.id }}"
+      allow_partial_import: "{{ dr_partial_import }}"
+      auth: "{{ ovirt_auth }}"
+      cluster_mappings: "{{ dr_cluster_map }}"
+      domain_mappings: "{{ dr_domain_map }}"
+      vnic_profile_mappings:  "{{ dr_network_map }}"
+      role_mappings: "{{ dr_role_map }}"
+  register: template_register_result
+
+- name: Append failed Template to issues failed_template_names
+  set_fact:
+    failed_template_names: "{{ failed_template_names }} + [ '{{ unreg_template.name }}' ]"
+  when: template_register_result | failed
+
+- name: append succeed_template_names
+  set_fact:
+    succeed_template_names: "{{ succeed_template_names }} + [ '{{ unreg_template.name }}' ]"
+  when: template_register_result | succeeded
```

Comment:
@machacekondra do you know if there is a way to avoid the copy paste of append_log